### PR TITLE
feat(AWS ActiveMQ): Support `maximumBatchingWindow`

### PR DESCRIPTION
**Pull request by rm-hull**
As per #10580, this PR adds `maximumBatchingWindow` property to serverless.yaml for ActiveMQ event sources

### CODE REVIEW

File: docs/providers/aws/events/activemq.md [hunk 1]
```diff
@@ -54,9 +54,10 @@ functions:
           basicAuthArn: arn:aws:secretsmanager:us-east-1:01234567890:secret:MySecret
 ```
 
-## Specifying batch size
+## Specifying batch size and batch window
```
Comment (pgrzesik) [inline docs/providers/aws/events/activemq.md:57]: let's reference that property in `serverless.yml.md` file

Comment (rm-hull) [reply docs/providers/aws/events/activemq.md:57]: Updated in 9f418fc917785ca522dd20f1c61169d3d8eee3ed
Note: I added it at the same position in `serverless.yaml.md` as #10591, so likely to be a merge conflict (both need to be accepted) ... i can fix that as soon as the other is merged though

File: docs/providers/aws/guide/serverless.yml.md [hunk 2]
```diff
@@ -928,6 +928,32 @@ functions:
...
+
+```yaml
+functions:
+  hello:
+    # ...
+    events:
+      - activemq:
+          # ARN of ActiveMQ Broker
+          arn: arn:aws:mq:us-east-1:0000:broker:ExampleMQBroker:b-xxx-xxx
+          # name of ActiveMQ queue consume from
```
Comment (pgrzesik) [inline docs/providers/aws/guide/serverless.yml.md:943]: let's write it as `Name of ActiveMQ queue to consume from`

File: docs/providers/aws/guide/serverless.yml.md [hunk 3]
```diff
@@ -928,6 +928,32 @@ functions:
...
+          queue: queue-name
+          # Secrets Manager ARN for credentials required to do basic auth
```
Comment (pgrzesik) [inline docs/providers/aws/guide/serverless.yml.md:945]: let's write it as `Secrets Manager ARN for basic auth credentials`

File: docs/providers/aws/guide/serverless.yml.md [hunk 4]
```diff
@@ -928,6 +928,32 @@ functions:
...
+          basicAuthArn: arn:aws:secretsmanager:us-east-1:01234567890:secret:MySecret
+          # Optional, must be in 1-10000 range
```
Comment (pgrzesik) [inline docs/providers/aws/guide/serverless.yml.md:947]: Let's be consistent with specifying ranges - below you're stating what is the max and min values and here you reference it as a range - is there a reason for that? I would stick to stating what is min and what is max value

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Thank you, it looks good, but I have some suggestions when it comes to wording, please let me know what do you think

File: docs/providers/aws/guide/serverless.yml.md (hunk 2 above)
Comment (rm-hull) [reply docs/providers/aws/guide/serverless.yml.md]: Fixed in 952ae3e404ad26c6331dd3f882ea7137efaeb031

File: docs/providers/aws/guide/serverless.yml.md (hunk 3 above)
Comment (rm-hull) [reply docs/providers/aws/guide/serverless.yml.md]: Fixed in 952ae3e404ad26c6331dd3f882ea7137efaeb031 #

File: docs/providers/aws/guide/serverless.yml.md (hunk 4 above)
Comment (rm-hull) [reply docs/providers/aws/guide/serverless.yml.md:947]: Fixed in 952ae3e404ad26c6331dd3f882ea7137efaeb031

Comment (rm-hull) [reply docs/providers/aws/guide/serverless.yml.md:947]: I'd copied it from the MSK example above, so will amend that too

**Review by pgrzesik — APPROVED** [review]
Thank you @rm-hull 🙇

**Pull request merged**